aiosmtpd/smtp.py

```diff
@@ -928,9 +936,10 @@ async def smtp_RCPT(self, arg: str) -> None:
             await self.push(syntaxerr)
             return
         address, params = self._getaddr(arg)
+        if address is None:
+            return await self.push("553 5.1.3 Error: malformed address")
         if not address:
-            await self.push(syntaxerr)
-            return
+            return await self.push(syntaxerr)
         if not self.session.extended_smtp and params:
             await self.push(syntaxerr)
             return
```

Comment:
Maybe make this style match the above (line 942)?

Reply:
That's an oopsie 😅 

Let me do some styling edits + commit + push.